### PR TITLE
Fix display problems

### LINKED ISSUE #207: [PE-D] Email too long cannot be seen
Email in view mode truncates too early when there's still space

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: tehkokhoe/ped#11

### PR DESCRIPTION
**Pull request by zihaowrez**
Fixes #207 #198 #174 #157 (and other similar issues if I missed)

### CODE REVIEW

**Review by kavxya — APPROVED** [review]
Tested and runs as specified.

**Review by cheehongw — COMMENTED** [review]
Why not replace the `Pane` with a `ScrollPane`?

File: src/main/java/seedu/address/ui/ContactDetailPanel.java [hunk 1]
```diff
@@ -21,10 +21,10 @@
     private final ObservableObjectValue<Person> observableSelectedPerson;
 
     @FXML
-    private Pane panel;
+    private VBox contactDetailPanel;
```
Comment (cheehongw) [inline src/main/java/seedu/address/ui/ContactDetailPanel.java:24]: Perhaps replace with `ScrollPane` here?

**Review by cheehongw — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/ui/ContactDetailPanel.java (hunk 1 above)
Comment (zihaowrez) [reply src/main/java/seedu/address/ui/ContactDetailPanel.java:24]: I tried `ScrollPane`, but I think `ListView` works better

**Review by cheehongw — COMMENTED** [review]
hmm I guess it's fine if we cannot replace with `ScrollPane`

**Review by cheehongw — APPROVED** [review]

**Comment by zihaowrez** [comment]
> hmm I guess it's fine if we cannot replace with `ScrollPane`

The problem with `ListView` is that when you click on the region, its background color changes. I tried to fix it by setting `contactDetailView.onMouseClicked(null)`, but it doesn't seem to work.

**Pull request closed without merging**